PythonRegression/util/threading_logic/pool_logic.py

```diff
@@ -6,7 +6,7 @@
 
 def start_pool(function,iterations,args):
     #Assigns the process worker pool to either the cpu count or the number of nodes for the call, whichever is lower
```

Comment:
you forgot to update the comment.
By the way you can change it to a doc string since it right below the function definition